Formula/node.rb

```diff
@@ -19,8 +19,8 @@ class Node < Formula
   # We track major/minor from upstream Node releases.
   # We will accept *important* npm patch releases when necessary.
```

Comment:
Could you highlight please how important this update is?
Does it fix any critical bugs or security vulnerabilities, for example?

Reply:
It should have been updated at the same time as Node 14.4.0 but wasn't.

Reply:
This not being merged _is_ causing an issue for me
```
➜ npm i -g npm
npm ERR! code ERR_OUT_OF_RANGE
npm ERR! The value of "err" is out of range. It must be a negative integer. Received 536870212
```